src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,151 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> prev;
+    private Node<T> next;
```

Comment:
it's better to call tail and head, or firstNode and lastNode. in code it's seems that you are operating not the head or tail, but the current prev or next

Reply:
we shouldn't use class Object.

Reply:
fixed